src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Instrumentation no longer store raw objects like `HttpWebRequest` in
```

Comment:
Lets be more explicit that this PR is fixing this for HttpClient in .NET Framework. HttpClient in .NET Core already had Enrich

Reply:
Added the following text: "Instrumentation for `HttpWebRequest`...."